Makefile

```diff
@@ -24,7 +25,7 @@ init-dep: init-dep-ui init-dep-api
 .PHONY: init-dep-ui
 init-dep-ui:
 	@echo "> Initializing UI dependencies ..."
-	@cd ${UI_PATH} && yarn
+	@cd ${UI_PATH} && yarn install --network-concurrency 1
```

Comment:
just curious why do we need this `--network-concurrency 1`

Reply:
It is related to this issue: https://github.com/yarnpkg/yarn/issues/7212

We are installing react-lazylog from our forked git repository: https://github.com/gojekfarm/react-lazylog and in it's package.json, we have `prepare` script that could raise "data race" during dependencies installation.